neo/neo.py

```diff
@@ -81,12 +81,11 @@ def main(
         github_base_ref: str,
         github_head_ref: str,
         include_regex: str,
-        defaults=None,
-        default_patterns=None,
+        defaults: bool = False,
+        default_patterns: list = None,
```

Comment:
```suggestion
        default_patterns: list | None = None,
```

The current way warns with:
`Type "None" cannot be assigned to type "list[Unknown]",
because "None" is not of type list :)